doc/messages/messages_list.rst

```diff
@@ -0,0 +1,498 @@
+.. _messages-list:
```

Comment:
This file is generated too, right ? Maybe we need to add it to the `.gitignore` ?

Reply:
We could, but I thought it would be good to show what the file somewhat looks like instead of only having the code.

Reply:
Do we need to think about updating it when creating a checker for example ? Or is it simply an example that will stay out of date and be replaced by sphinx ?  If we need to update it's more work, if we do not update it could be confusing. Maybe being able to look at the (previous) result in github pages is good enough ?

Reply:
We could update it but it shouldn't be necessary. This is similar to `doc/technical_reference/extensions.rst` which can be outdated as well.